### PR TITLE
[Pytorch] Added squared ReLU implementation

### PR DESCRIPTION
**Pull request by phu0ngng**
## Type of change

- [ ] Documentation change (change only to the documentation, either a fix or a new content)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# Checklist:

- [x] I have read and followed the [contributing guidelines](https://github.com/NVIDIA/TransformerEngine/blob/main/CONTRIBUTING.rst)
- [x] The functionality is complete
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: transformer_engine/pytorch/te_onnx_extensions.py [hunk 1]
```diff
@@ -166,6 +166,16 @@ def onnx_fp8_relu(g, inputs, scale, amax, scale_inv, fp8_tensor, otype):
     return relu
 
 
+@symbolic_helper.parse_args("v", "v", "v", "fs", "i", "i")
+def onnx_fp8_srelu(g, inputs, scale, amax, scale_inv, fp8_tensor, otype):
+    """ONNX graph for fp8_srelu"""
+    # pylint: disable=unused-argument
+    srelu = compute_in_fp32(g, inputs, torch.onnx.symbolic_opset9.srelu)
```
Comment (ksivaman) [inline transformer_engine/pytorch/te_onnx_extensions.py:173]: I don't think this exists: https://github.com/pytorch/pytorch/blob/main/torch/onnx/symbolic_opset9.py

Comment (phu0ngng) [reply transformer_engine/pytorch/te_onnx_extensions.py]: I remove srelu from onnx and it tests. Thanks

**Review by ksivaman — APPROVED** [review]
LGTM

**Review by timmoon10 — APPROVED** [review]

**Pull request merged**